docs/content/guides/rows/rows-sorting.md

```diff
@@ -34,16 +32,14 @@ searchCategory: Guides
 
 # Rows sorting
 
-Sort data alphabetically or numerically, in ascending, descending or a custom order, by one or
-multiple columns.
+Sort data alphabetically or numerically, in ascending, descending or a custom order, by one or multiple columns.
 
 [[toc]]
 
 ## Overview
 
-With sorting, you can easily rearrange rows of data, based on the values in specific columns. This
-is particularly useful for analyzing and organizing large data sets, which helps you identify
-patterns and trends.
+With sorting, you can easily rearrange rows of data, based on the values in specific columns. This is particularly useful for analyzing and organizing large
+data sets, which helps you identify patterns and trends.
```

Comment:
Hello, I would propose the following version for consideration:

"With the use of sorting, the user has the ability to easily rearrrange the rows of data, based on the values in specific columns. This functionality is particularly useful in terms of analysis and organization of the large data sets, which support the user in the identification of patterns and trends."